lexicons/app/bsky/contact/getMatches.json

```diff
@@ -36,8 +36,19 @@
       },
       "errors": [
         {
-          "name": "TODO",
-          "description": "TODO"
+          "name": "INVALID_DID"
```

Comment:
In practice, this error shouldn't happen, as the appview should never send to rolodex an invalid DID as the user session, but that's fine to keep here.

Or we could do a per-route way of checking this. Because it will be repeated for many of these endpoints, and it doesn't necessarily make sense for any of them.